worlds/stardew_valley/__init__.py

```diff
@@ -278,7 +278,8 @@ def fill_slot_data(self) -> Dict[str, Any]:
 
         excluded_options = [BundleRandomization, BundlePrice,
```

Comment:
```suggestion
        excluded_options = [BundleRandomization.internal_name, BundlePrice.internal_name,
                            NumberOfMovementBuffs.internal_name, NumberOfLuckBuffs.internal_name]
```
and remove the unnecessary pops after.